20/alpine/alpine-rebar3/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM erlang:20.3.8.9-alpine
```

Comment:
should we not use the exact version ? (because that will change too often) , use `20.3` is better than the fully exact version?